### DISCUSSION TITLE
div class from giscuss gets cut off

### DISCUSSION THREAD

**Discussion by Andre601** [giscus/giscus#370] [Q&A] [answered]
Just noticed that a pop-up bubble from Giscus, which tells you to log in in order to react is cut off.

Unfortunately can I not really find out if that is an issue on Giscus' end or if the page builder (MkDocs) and/or theme (Material for MkDocs) does somehow do this.

I do use a custom CSS for giscus, but other than setting the theme to dark and putting the text box at the top is there no difference to the available themes it uses.
CSS can be found [here](https://github.com/Andre601/blog/blob/master/docs/assets/css/style/giscus.css)

Here's an example of what happens:
![image](https://user-images.githubusercontent.com/11576465/150772537-04d2518f-741a-46a8-8f54-7effd9950951.png)

I hope someone can help debug this.
The page where you can see this behaviour in question is [this one here](https://www.andre601.ch/blog/posts/2021/10-20-lets-talk-about-verification/).

I posted this on Material for MkDocs' repository too in the hope of help.

**Comment by laymonage** [comment]
This is the downside of having the comments box above the comments instead of below. The reaction button pops up downwards. Since it uses `absolute` positioning, it won't contribute to the widget's page height, so the iframe won't be resized when the pop-up opens.

One workaround is to add a bottom margin, e.g. add

```css
.gsc-timeline {
  margin-bottom: 8rem;
}
```

to your `giscus.css` file. Of course, this would add an unnecessary margin when there are no comments yet.

**Reply by Andre601** [reply]
Maybe a future idea could be a "jump to textbox" solution?
Like the current issue with the default comment ordering is the fact that the text box will be all the way at the bottom of the page, requiring you to scroll completely down, which is just tedious.

Another solution could perhaps be to only load a certain amount of comments and have the remaining one behind a "expand" button, similar to how GitHub hides entries in a PR if there are too many...

Either way, I'll implement this solution and let you know if it worked.

**Reply by laymonage** [reply]
> Maybe a future idea could be a "jump to textbox" solution? Like the current issue with the default comment ordering is the fact that the text box will be all the way at the bottom of the page, requiring you to scroll completely down, which is just tedious.

I don't think this is possible as the page is loaded within an iframe. Scrolling is controlled by the parent page.

> Another solution could perhaps be to only load a certain amount of comments and have the remaining one behind a "expand" button, similar to how GitHub hides entries in a PR if there are too many...

This is already the case.

**Comment by laymonage** [comment, marked as answer]
Fixed in #449.
